air/src/proof/ood_frame.rs

```diff
@@ -67,26 +64,15 @@ impl OodFrame {
     {
         assert!(self.trace_states.is_empty(), "trace sates have already been set");
 
-        // save the evaluations of the current and then next evaluations for each polynomial
-        let (main_and_aux_trace_states, lagrange_trace_states) = trace_ood_frame.to_trace_states();
+        // save the evaluations with the current and next evaluations interleaved for each polynomial
+        let main_and_aux_trace_states = trace_ood_frame.to_trace_states();
```

Comment:
I don't think the comment here should mentioned "interleaved".

Reply:
Indeed, fixed now